db-tools-i18n/src/main/java/com/splicemachine/db/iapi/tools/i18n/LocalizedResource.java

```diff
@@ -362,13 +376,13 @@ else if (type == Types.TIMESTAMP ) {
 		}
 
 	public String getDateAsString(Date d){
-		if (!enableLocalized){
+		if (!customFormatDate){
```

Comment:
enableLocalization can also be called in ij.java for the LOCALIZEDDISPLAY option, so perhaps we still need to check enableLocalized:

> if (!enableLocalized && !customFormatDate){

Same comment for getTimeAsString and  getTimestampAsString

Reply:
when `enableLocalization(boolean mode)` is called it is calling `getInstance().init();`, which will set `customFormatDate` correctly, see this code in lines 168-178:

```
public void init (String encStr, String locStr, String msgF){
  // ...
	if (enableLocalized && locale != null){
		formatDecimal = (DecimalFormat)DecimalFormat.getInstance(locale);
		formatNumber = NumberFormat.getInstance(locale);
		formatDate = DateFormat.getDateInstance(DateFormat.LONG,locale);
		formatTime = DateFormat.getTimeInstance(DateFormat.LONG,locale);
		formatTimestamp = DateFormat.getDateTimeInstance(DateFormat.LONG,
				DateFormat.LONG, locale);
		customFormatDate = true;
		customFormatTime = true;
		customFormatTimestamp = true;
	}
   //...
}
```

but i forgot to reset to false.

Reply:
done